tests/testthat/_snaps/to_md/to_md-works-for-Rmd.Rmd

```diff
@@ -33,7 +33,7 @@ plot(pressure)
 
 Non-RMarkdown blocks are also considered
 
-```{julia, info=bash}
+```{julia}
```

Comment:
oops

Reply:
actually I am not sure why it used to be "info=bash"?

Reply:
I just found why:

I was trying to test the integrity of the object (e.g. that it was copied before writing instead of transformed in place). I did not consider the fact that I was overwriting a markdown code block within the chunks:

https://github.com/ropensci/tinkr/blob/60d35732c24dcd3cd4761055ec8061b97a55fd43/tests/testthat/test-to_md.R#L98-L110

That being said, I think the behaviour of having the 'info' propagate does actually make sense because it's not unheard of to have custom chunk options.

Reply:
I don't get why my code changed this, though.

Reply:
@zkamvar why was there originally the "info=bash"?